holoviews/core/data/iris.py

```diff
@@ -4,6 +4,9 @@
 from itertools import product
 
 import iris
+from iris.coords import DimCoord
+from iris.cube import CubeList
+from iris.experimental.equalise_cubes import equalise_attributes
```

Comment:
Will be good to have the iris interface moved to geoviews. Could this be done for 1.10.6?

Reply:
Tests need to be moved into the holoviews package first.

Reply:
You mean 'geoviews' package?

Reply:
No I mean, the ``/tests`` need to move to ``/holoviews/tests``.

Reply:
The interface tests are defined as mix-in classes, so if I want to run them in geoviews I have to be able to import them from holoviews. We also promised this to the bokeh folks so they can run our bokeh unit tests easily.